### PR TITLE
Feature "cyclomatic complexity" in coding-standards.md

### PR DESCRIPTION
**Pull request by GitoTek**
- [x] : Added pictures for the cyclomatic-complexity section in coding-standards.md
- [x] : Added folders & subfolders for the pictures ("/readme-content/pictures/coding-standards/cyclomatic-complexity")
- [x] : Added content for the cyclomatic-complexity section in coding-standards.md

### CODE REVIEW

File: readme-content/coding-standards.md [hunk 1]
```diff
@@ -9,4 +9,36 @@
 * [Our project retrospective](/readme-content/project-retrospective.md)
 
 
-To do...
\ No newline at end of file
+## Work with cyclomatic complexity <br>
```
Comment (CedricCepeda-Epitech) [inline readme-content/coding-standards.md:121]: Add section and sub-section in the table of content

**Comment by CedricCepeda-Epitech** [comment]
Good explanation, just need to add the link in the table of content and everything is good for me

**Comment by GitoTek** [comment]
> Good explanation, just need to add the link in the table of content and everything is good for me

I will work on it as soon as possible.

**Comment by CedricCepeda-Epitech** [comment]
Why didn't you add the other titles like this one on the table of content ?
image

**Comment by GitoTek** [comment]
> Why didn't you add the other titles like this one on the table of content ? image

A mistak of my own, it is fixed now.

**Review by Colimaters — APPROVED** [review]
All good !

**Review by CedricCepeda-Epitech — APPROVED** [review]

**Pull request merged**